model-optimizer/extensions/middle/MXFFTToDFT.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.dft import DFT, IDFT
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.pad import Pad
+from mo.ops.reshape import Reshape
+from mo.ops.squeeze import Squeeze
+from mo.ops.strided_slice import StridedSlice
+from mo.ops.unsqueeze import Unsqueeze
+
+
+class MXFFTToDFT(MiddleReplacementPattern):
+    """
+    This transformation converts the operation MXFFT into OpenVINO DFT (if the attribute 'is_inverse' is False),
+    or into OpenVINO IDFT (otherwise).
+    """
+    enabled = True
+
+    def find_and_replace_pattern(self, graph: Graph):
```

Comment:
Do we have an mxnet layer test which shows us that the inference results are the same?

Reply:
There is some problem with such tests: MxNet layers `FFT` and `IFFT` are supported only in GPU, and **MxNet must be compiled with CUDA support**. But, if we install MxNet using `pip3 install mxnet`, MxNet generic will be installed, and this generic MxNet is not compiled with CUDA support. And I have no machines with CUDA support (it is NVIDIA technology, and I have no machine with NVIDIA GPU).

If I run example
```python3
import numpy as np
import mxnet as mx

data = np.random.normal(0,1,(3,4))
out = mx.contrib.ndarray.fft(data = mx.nd.array(data,ctx = mx.gpu(0)))
```
in Windows 10 64-bit or Ubuntu 18.10 64-bit, I get 
```sh
mxnet.base.MXNetError: [10:19:52] src/storage/storage.cc:119: Compile with USE_CUDA=1 to enable GPU usage
```
Similar error in Debian 10 64-bit.

And more, I can write such layer tests, but these tests will be failed on any machine without NVIDIA GPU.

Hence, we can use only MxNet `FFT` and `IFFT` documentation.